float32 north
float32 east
float32 radius
float32 height
